pyproject.toml

```diff
@@ -9,3 +9,56 @@ requires = ["setuptools", "wheel",
     "numpy ~= 1.21.0 ; python_version == '3.9' and platform_system == 'Darwin' and platform_machine == 'arm64'",
     "numpy ~= 1.21.0 ; python_version >= '3.10'",
     ]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "spacepy"
+version = "0.5.0a0"
+description = "SpacePy: Tools for Space Science Applications"
+readme = "README.md"
+requires-python = ">=3.6"
+license = { "file" = "LICENSE.md" }
+authors = [{ "name" = "SpacePy team", "email" = "spacepy@lanl.gov" }]
```

Comment:
This email doesn't exist any more, AFAIK. It may not be a hold-up for this, but we should think about what the right replacement is.

Reply:
Yeah, I think we've been "thinking" about that for quite some time and haven't come up with a great idea yet. I think we can do name/value pairs and just put in the four active devs if it comes to it.

Reply:
Well, let's put in an issue for it and then we can figure out what we want to do. For now I'll mark as resolved.